.github/workflows/try.yml

```
@@ -4,47 +4,31 @@ name: CI
 on:
  
   push:
-    branches: [ "master" ]
+    branches: [ "main","better_workflow" ]
```

Remove `"better_workflow"` from here:
```suggestion
    branches: [ "main" ]
```

---

In any case, we will remove it altogether in the next PR